server/db-pg/model.js

```diff
@@ -0,0 +1,73 @@
+const { Client } = require('pg');
+require('dotenv').config();
+
+const database = process.env.DATABASE_NAME;
+
+const client = new Client({
+  database,
+});
+
+client.connect();
+
+// This modifies the given restaurant object by parsing its `hours` and
+// `location` values, which are stored as stringified JSON in the database.
+const parseRestaurant = (restaurant) => {
+  restaurant.location = JSON.parse(restaurant.location || '""');
+  restaurant.hours = JSON.parse(restaurant.hours || '""');
+  return restaurant;
+};
+
+// Same as `pasreRestaurant`, except it stringifies the relevant values.
+const stringifyRestaurant = (restaurant) => {
+  restaurant.location = JSON.stringify(restaurant.location || '');
+  restaurant.hours = JSON.stringify(restaurant.hours || '');
+  return restaurant;
+};
+
+const getRestaurant = (id) => {
+  const queryString = `select * from restaurants where id = ${id}`;
+
+  return client.query(queryString)
+    .then(res => parseRestaurant(res.rows[0]));
+};
+
+const getRestaurantByName = (name) => {
+  const queryString = `select * from restaurants where name = '${name}'`;
+
+  return client.query(queryString)
+    .then(res => parseRestaurant(res.rows[0]));
+};
+
+const deleteRestaurant = (id) => {
+  const queryString = `delete from restaurants where id = ${id} returning *`;
```

Comment:
what are you thoughts on query commands in allcaps? 
ie. 
```SQL
DELETE FROM restaurants WHERE id = ${id} RETURNING *;
```
These seem a bit more readable personally, but I imagine that's up for debate.

Reply:
I think I'm going to leave it as is for now, because there would be a lot of changes to make, but you're probably right. The allcaps is at the very least a common practice, so it's probably good to follow it.